src/Modules/Roles/roleService.js

```diff
@@ -0,0 +1,43 @@
+const Role = require("./roleModel");
+const DbService = require("../../Service/DbService");
+const serviceHandler = require("../../Utils/serviceHandler");
+const CustomError = require("../../Errors/CustomError");
+
+const model = new DbService(Role);
+
+const roleService = {
+  create: serviceHandler(async (data) => {
+      return await model.save(data);
+  }),
+
+  getAll: serviceHandler(async (data) => {
+   const savedData=await model.getAllDocuments({isDelete:false})
+   const totalCount=await model.totalCounts({isDelete:false})
+   return{savedData,totalCount}
```

Comment:
getAllDocuments accept two parameters, one for query , second for customObject for limit, sort and other features, 
You are expecting 'data' parameter but not using it anywhere, pass it as a second parameter in getAllDocuments method